src/command_modules/azure-cli-ams/HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+0.5.1
++++++
+* Bug fix for asset and account filters update.
+* Minor pylint changes
```

Comment:
Pylint changes are not customer facing and should not be in history.

Reply:
Removed pylint comment.